server/src/apiClient/TheMovieDataBaseClient.js

```diff
@@ -0,0 +1,29 @@
+import got from "got"
+import dotenv from "dotenv"
+dotenv.config()
+
+const tmdbApiToken = process.env.TMDB_API_ACCESS_TOKEN
```

Comment:
should use the pattern from Interacting with a Third Party API where all process.envs are set up in the config